### DISCUSSION TITLE
Metadata correction

### DISCUSSION THREAD

**Discussion by vodobaas1** [immich-app/immich#12581] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The duplication utility is great and I've been using that a lot.  In the same vein as the duplication checking, could there be a method to look at pictures that have the same hash or what not and suggest fixing the metadata for the pictures?  What I mean is, I have a picture and I send it to my SO, before Immich of course and so she has a different data from me sending it over text than I do.  Because we keep our photos somewhat separate I don't want to get rid of the dup, but I would like to get the metadata corrected.  I also have a bunch of photos that would have come over from whatsapp that are all messed up and all have one date.  

### Platform

- [x] Server
- [ ] Web
- [ ] Mobile

**Comment by voitekhovich** [comment]
Immich should implement functions for editing metadata and saving it in the original file. There is a lot of archive data that does not contain metadata. Especially creating a photo or recording a video. Within Immich, editing the date works, but if you unload files, or need to reconfigure Immich from scratch, the information will be lost and you will have to configure everything again.
